src/pages/Profile.tsx

```diff
@@ -0,0 +1,266 @@
+import { PageContainer } from './components/PageContainer'
+
+import { TEXT_KEYS, TEXTS } from 'src/resources/texts'
+import SlackIcon from '../resources/slack-icon.svg'
+import { useTranslation } from 'react-i18next'
+// import GapsPage from './GapsPage'
+// import SingleLineMapPage from './SingleLineMapPage'
+import { SearchContext } from '../model/pageState'
+import LineNumberSelector from './components/LineSelector'
+import OperatorSelector from './components/OperatorSelector'
+import RouteSelector from './components/RouteSelector'
+
+// time inputs
+import { DateSelector } from './components/DateSelector'
+import MinuteSelector from './components/MinuteSelector'
+import { TimeSelector } from './components/TimeSelector'
+import moment from 'moment'
+import { useDate } from './components/DateTimePicker'
+
+// GRAPH
+import ArrivalByTimeChart from 'src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart'
+import { GroupByRes, useGroupBy } from 'src/api/groupByService'
+
+
+
+
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts'
+
+const Profile = () => {
+    return  (
+        <GeneralDetailsAboutLine />
+    )
+}
+
+const GeneralDetailsAboutLine = () => {
+    const { t } = useTranslation()
+
+    const { search, setSearch } = useContext(SearchContext)
+    const { operatorId, lineNumber, timestamp, routes, routeKey } = search
+  
+    return (
+      <>
+      <PageContainer className="line-data-container">
+        {/* choose operator */}
+        <Grid xs={8}>
+          <OperatorSelector
+            operatorId={operatorId}
+            setOperatorId={(id) => setSearch((current) => ({ ...current, operatorId: id }))}
+          />
+        </Grid>
+
+        {/* choose line number */}
+        <Grid xs={8}>
+          <LineNumberSelector
+            lineNumber={lineNumber}
+            setLineNumber={(number) => setSearch((current) => ({ ...current, lineNumber: number }))}
+          />
+        </Grid>
+
+        {/* choose route*/}
+        <Grid xs={12}>
+            {routes &&
+              (routes.length === 0 ? (
+                <NotFound>{TEXTS.line_not_found}</NotFound>
+              ) : (
+                <RouteSelector
+                  routes={routes}
+                  routeKey={routeKey}
+                  setRouteKey={(key) => setSearch((current) => ({ ...current, routeKey: key }))}
+                />
+              ))}
+          </Grid>
+        </PageContainer>
+
+      <LineProfileComponent />
+      <LineGraphSchedule />
+      </>
+    )
+  }
+
+
+const LineProfileComponent = () => {
+  const { search, setSearch } = useContext(SearchContext)
+  const { operatorId, lineNumber, timestamp, routes, routeKey } = search
+  const [agencyList, setAgencyList] = useState<Agency[]>([])
+
+  return (
+    <Grid xs={12} lg={6}>
+      <div className="widget">
+        <h2 className="title">{TEXTS.profile_page}</h2>
```

Comment:
```suggestion
        <h2 className="title">{t('profile_page')}</h2>
```

Reply:
text should be from: https://github.com/hasadna/open-bus-map-search/blob/33e83cf2747b10c6d86dfbcb23fb368427dbbabd/src/locale/en.json#L4